faqs/risk-management.md

```diff
@@ -0,0 +1,116 @@
+
+## What is the connection between a Risk Construct and voting in new collateral?
+
+Proposed new collateral will go through a due-diligence process, including a qualitative and quantitative assessment. Since every asset has differences, it is important to evaluate each of them individually according to a well-defined collateral assessment process. MKR Holders' weightings of these assessments will depend on which Risk Construct is used.  
+
+## What is Qualitative Risk Analysis, and how should one approach it?
+
+Qualitative Risk Analysis uses a relative or descriptive scale to measure risks based on their probability of occurring and the impact on the asset should they occur. To evaluate an asset for its risks, one needs to understand and assess the organization behind the asset, the characteristics of the token, and other relevant details. This is called a Qualitative assessment. The process of compiling the information has three parts:
+
+*   The Initial Collateral Onboarding Process: this covers the trade support structure, distribution of token holdings and available data series.
+
+*   The Operational Assessment Process: this covers the organization itself, the organization's operational risks, regulatory risks, the quality of the community, etc.
+
+*   The Technological Assessment Process: this covers the robustness and security of the underlying technology.
+
+Due diligence related to each of these three parts will give one information that can be compiled into distinct features and assigned ratings. These features are:
+
+*   Team; Core team and advisors
+
+*   Community; Sentiment analysis
+
+*   Technology ;  Security and completeness review
+
+*   Market and Competitiveness;  SWOT analysis
+
+*   Business Models ; Structural and legal analysis
+
+An overall score, formed as a result of the individual ratings, will serve as the mathematical input for the Risk Model. The score, judged against other assets in the portfolio, will be the adjustment factor to the risk parameters of that token.
+
+## What is Exposure Risk?
+
+Exposure Risk refers to the exposure one has to particular collateral and generally whether one has too much.  Exposure Risk is relative to the available supply of the asset and its liquidity and quality rating. For a CDP, if a significant portion of an asset's supply was locked as collateral, its subsequent liquidation could result in significant price impact on the market. The risk to the MakerDAO system is that a price drop of that collateral type may result in the liquidation of increasingly more of the underlying asset in CDPs than the market can handle.
+
+Exposure risk also takes into account the Free Float of the token and its relationship to the token's liquidity. Free Float is the readily traded portion of the available supply. Liquidity in the market is a result of the free float.
```

Comment:
inconsistent capitalization: Exposure risk. Please double check the rest of the doc.